Docs/bugs rephrasing hacakathon (#79)
neicnordic/neic-sda: docs/structure.md

Deployment related choices
--------------------------

### Federated vs stand-alone

In a Federated setup, the `FederatedEGA` archive node setup locally need to exchange status updates with the `CentralEGA` in a synchronized manner to basically orchestrate two parallel processes

1. The multi-step process of uploading and safely archiving encrypted files holding both sensitive phenome and genome data.

2. The process of the Submitter annotating the archived data in an online portal at `CentralEGA`, resulting in assigned accession numbers for items such as DataSet, Study, Files etc.

In a stand-alone setup, the deployed service has less remote synchronisation to worry about, but on the other hand more components might be required (e.g ([orchestrator](services/orchestrator))) to also handle annotations/meta-data locally, as well as to deal with identifiers etc.

> The NeIC SDA is targeting both types of setup, to open up the use of the re-use of components to more use cases than initially envisioned.

Reviewer: ```suggestion
The NeIC SDA is targeting both types of setup but also to allow for the possibility to re-use components in more use cases than initially envisioned.
```
